### DISCUSSION TITLE
Using Python libraries downloaded from pip

### DISCUSSION THREAD

**Discussion by Electric-Dragon** [pybricks/support#784] [Q&A]
Hello,
I want to use a library I downloaded through pip and a custom file along with my code. Is there a way I can achieve that?

The library is used to use some functionalities of the custom file.

**Comment by laurensvalk** [comment]
In https://beta.pybricks.com/, you can now make multiple files and import between them.

There isn't currently a way to download from PyPI within our editor automatically. Upstream MicroPython is working on this kind of functionality though, so perhaps we can use some of that in Pybricks in the future.

It might possible to make a local project and run it on the hub via https://github.com/pybricks/pybricksdev.
